core/toolbox.js

```diff
@@ -533,7 +533,9 @@ Blockly.Toolbox.TreeNode = function(toolbox, html, opt_config, opt_domHelper) {
   goog.ui.tree.TreeNode.call(this, html, opt_config, opt_domHelper);
```

Comment:
Certainly don't mind general questions! 
I'm not sure I understand what you're asking though since I also only know enough js to get me in trouble... but were you expecting this constructor to be defined as Blockly.Toolbox.prototype.TreeNode = function(...)? I don't know what happens if you try and make a constructor on a prototype like that. Is that possible?